test/surface_test.py

```diff
@@ -284,8 +309,27 @@ def test_subsurface_rle2(self):
         s2 = s1.subsurface(sub_rect)
         s2.fill(color2)
         s0.blit(s2, (0, 0))
-        self.assertTrue(s1.get_flags() & pygame.RLEACCEL)
-        self.assertTrue(not s2.get_flags() & pygame.RLEACCEL)
+        self.assertTrue(s1.get_flags() & pygame.RLEACCELOK)
+        self.assertTrue(not s2.get_flags() & pygame.RLEACCELOK)
+
+    def test_solarwolf_rle_usage(self):
+
+        def optimize(img):
+            clear = img.get_colorkey()
+            img.set_colorkey(clear, RLEACCEL)
+            return img.convert()
+
+        pygame.display.init()
+        try:
+            window_surf = pygame.display.set_mode((640, 480))
+
+            image = pygame.image.load(example_path(os.path.join("data",
+                                                        "ship-up-boost1.png")))
+            image = optimize(image)
```

Comment:
Is this line duplicated on purpose?

Reply:
Yes, calling it twice was what caused the crash in the other PR once I stripped away all the other unessential parts of the test.

Reply:
If it's not supposed to be duplicated, the optimize function could be removed.